streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -273,6 +273,7 @@ private Properties prepareTest() throws Exception {
         streamsConfiguration.put(ConsumerConfig.HEARTBEAT_INTERVAL_MS_CONFIG, 100);
         streamsConfiguration.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "" + STREAMS_CONSUMER_TIMEOUT);
         streamsConfiguration.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        streamsConfiguration.put("internal.leave.group.on.close", true);
```

Comment:
Would it make sense to have a constant for this in some Streams internal utils class? It seems like a typo in any of these tests won't be caught by a test failure, right?